.github/workflows/iroha2-dev-pr-title.yml

```
@@ -9,8 +9,40 @@ jobs:
   check_pr_title:
     runs-on: ubuntu-latest
     steps:
-      - name: Check PR title format
-        env:
-          PR_TITLE: ${{ github.event.pull_request.title }}
-        # Example: "[feature] #1: Some new feature"
-        run: echo $PR_TITLE | grep -Eq "^\[(feature|fix|refactor)\] \#[[:digit:]]+:.+$"
+      - uses: actions-ecosystem/action-regex-match@v2
+        id: feature-match
+        with:
+          text: ${{ github.event.pull_request.title }}
+          regex: '^\[(feature|feat\.?)\] \#?([[:digit:]]+|\(([[:digit:]]+,?\s?)+\)):.+$'
+
+      - uses: actions-ecosystem/action-regex-match@v2
+        id: refactor-match
+        if: ${{ steps.feature-match.outputs.match == '' }}
+        with:
+          text: ${{ github.event.pull_request.title }}
+          regex: '^\[ref(actor)?\]( \#?([[:digit:]]+|\(([[:digit:]]+,?\s?)+\)):)?.+$'
+
+      - uses: actions-ecosystem/action-regex-match@v2
+        id: fix-match
+        if: ${{ steps.refactor-match.outputs.match == '' }}
+        with:
+          text: ${{ github.event.pull_request.title }}
+          regex: '^\[(bug)?fix(up)?\] \#?([[:digit:]]+|\(([[:digit:]]+,?\s?)+\)):.+$'
```

Example:
`[fix] #0, #1234, #56789: Title`

```suggestion
          regex: '^\[fix\] #\d+(, #\d+)*: .+$'
```

and the same for other categories.
According to [my attempt](https://github.com/s8sato/iroha/runs/4404431957?check_suite_focus=true#step:9:14) `#` works and `[[:digit:]]` does not works.

What is the point of leaving ambiguity?